### PR TITLE
Allows admins to set a custom tip of the round

### PR DESCRIPTION
**Pull request by DominikPanic**
Also fixed da code.

Fixes: #2237

### CODE REVIEW

File: code/__HELPERS/global_lists.dm [hunk 1]
```diff
@@ -1,8 +1,8 @@
-GLOBAL_LIST_EMPTY(metatips)
-GLOBAL_LIST_EMPTY(marinetips)
-GLOBAL_LIST_EMPTY(xenotips)
-GLOBAL_LIST_EMPTY(joketips)
-#define ALLTIPS (GLOB.marinetips + GLOB.xenotips + GLOB.joketips + GLOB.metatips)
+GLOBAL_LIST_INIT(metatips, file2list("strings/tips/marine.txt"))
```
Comment (spookydonut) [inline code/__HELPERS/global_lists.dm]: was done this way to avoid world load lag, not sure why its being changed back

Comment (DominikPanic) [reply code/__HELPERS/global_lists.dm]: All the other strings get done this way, just like datum ref lists.

Comment (spookydonut) [reply code/__HELPERS/global_lists.dm]: datum ref lists is called during world/new this is before world/new

Comment (spookydonut) [reply code/__HELPERS/global_lists.dm]: ideally these should be loaded by a string subsystem init

Comment (Rohesie) [reply code/__HELPERS/global_lists.dm]: Check explanation for the change request.

Comment (DominikPanic) [reply code/__HELPERS/global_lists.dm]: It should be done via init ,this is such a tiny little margin that it is out of scope to request an entirely new subsystem. Currently this fixes more issues than it presents.

Comment (spookydonut) [reply code/__HELPERS/global_lists.dm]: this isnt a fix pr its a feature pr.  its not much effort to make a subsystem

Comment (spookydonut) [reply code/__HELPERS/global_lists.dm]: especially given these arent needed prior to round start they can be late loaded, i've been trying as much as possible to shift stuff from before/during world/New() to subsystem init to reduce loading lock ups.

Comment (DominikPanic) [reply code/__HELPERS/global_lists.dm]: the fortune cookies pick from the list, the previous implementation was broken, this is a fix PR too.

Comment (Rohesie) [reply code/__HELPERS/global_lists.dm]: This is a fix PR, not a features one. Nor a fix for his own feature.
Requesting for a subsystem is way out of scope.

File: code/__HELPERS/global_lists.dm [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
+GLOBAL_LIST_INIT(marinetips, file2list("strings/tips/xeno.txt"))
+GLOBAL_LIST_INIT(xenotips, file2list("strings/tips/meta.txt"))
```
Comment (spookydonut) [inline code/__HELPERS/global_lists.dm]: okay here we go, change these back to empty

File: code/controllers/subsystem/ticker.dm [hunk 3]
```diff
@@ -304,14 +305,11 @@ SUBSYSTEM_DEF(ticker)
...
-	GLOB.xenotips = world.file2list("strings/tips/xeno.txt")
-	GLOB.metatips = world.file2list("strings/tips/meta.txt")
-	GLOB.joketips = world.file2list("strings/tips/meme.txt")
 
-
-	if(prob(95) && length(ALLTIPS))
-		tip = pick(ALLTIPS)
+	if(selected_tip)
+		tip = selected_tip
+	else if(prob(95) && length(GLOB.alltips))
```
Comment (spookydonut) [inline code/controllers/subsystem/ticker.dm]:
```suggestion
	else
		GLOB.marinetips = world.file2list("strings/tips/marine.txt")
		GLOB.xenotips = world.file2list("strings/tips/xeno.txt")
		GLOB.metatips = world.file2list("strings/tips/meta.txt")
		GLOB.joketips = world.file2list("strings/tips/meme.txt")
		
		if(prob(95) && length(GLOB.alltips))
```

Comment (spookydonut) [reply code/controllers/subsystem/ticker.dm]: we only need to load these if you dont select a tip

**Comment by DominikPanic** [comment]
I'll wait for your PR to be merged first.

**Pull request merged**